src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Drawable {
+

```

Comment:
```suggestion

```

Reply:
Methods declarations must be separated from the previous code with an empty line

Reply:
Actually nobody does that, I mean in abstract classes (even in real one) there are no sense to separate method decration from class declaration by an empty line. Here is the examples of one method classes ( https://www.tutorialspoint.com/class-declaration-with-one-method-in-java, https://stackoverflow.com/questions/3670271/design-question-java-class-with-single-method-ok ...) You can google more examples by yourself, I never saw such approach as you did on real code. So please remove line breake